test/cpp/modules/manual/GmmTests.cpp

```diff
@@ -2,7 +2,7 @@
 #include "../../../../src/cpp/runner/ModuleLoader.h"
```

Comment:
omit ../../src/cpp in middle

Reply:
Can't do that - this file lies in the /test/cpp/... folder, not in /src/cpp/...